manifest-referrers-api.md

```diff
@@ -0,0 +1,109 @@
+# Manifest Referrers API
+
+[OCI artifact-manifest](./artifact-manifest.md) provides the ability to reference artifacts to existing artifacts. Reference artifacts include Notary v2 signatures, SBoMs and many other types. Artifacts that reference other artifacts SHOULD NOT be tagged, as they are considered enhancements to the artifacts they reference. To discover referenced artifacts a manifest referrers API is provided. An artifact client, such as a Notary v2 client would parse the returned manifests, determining which manifest type they will pull and process.
+
+The `referrers` API returns all artifacts that have a `subjectManifest` to given manifest digest. Referenced artifact requests are scoped to a repository, ensuring access rights for the repository can be used as authorization for the referenced artifacts.
```

Comment:
Sounds like this means that image indexes that reference an image will NOT be returned when a call to `/referrers` is made. I think it's probably worth adding an explicit note saying that this API would not return image indexes that refer to an image or images that refer to a layer.

Reply:
> this API would not return image indexes that refer to an image or images that refer to a layer

Seems to me like it would be a more useful API if it _did_...

Reply:
We scoped phase 1 to support signatures on indexes, as well as other reference types. For instance, you can sign an index, or an sbom. You'd pass in the digest of the index or sbom to get the signatures of that manifest.  
But, we hadn't considered the index as a reverse index of a manifest. It's a fair point and likey a scoping question whether we believe we need this for phase 1, vs. think about this in phase 2 to provide more time for how we'd really like to land this.